include/ck_tile/ops/gemm/kernel/gemm_tile_partitioner.hpp

```diff
@@ -35,4 +35,35 @@ struct GemmTilePartitioner
         return make_tuple(iM, iN);
     }
 };
+
+template <typename BlockGemmShape_>
+struct GemmTile1DPartitioner
```

Comment:
This should be renamed. Currently this class is serving multiple purposes. You should have one class for getting 1D mapping and the other one which is kind a OffsetedTilePartitioner like this one: https://github.com/ROCm/composable_kernel/blob/develop/include/ck/tensor_operation/gpu/grid/block_to_ctile_map.hpp#L866

Reply:
This should be updated in following PR